src/GitInfo/build/GitInfo.targets

```diff
@@ -496,7 +496,7 @@
 			DependsOnTargets="_GitRoot"
 			Condition="'$(GitRoot)' != '' And '$(GitCommit)' == ''">
 
-    <Exec Command='$(GitExe) -c log.showSignature=false log --format=format:$(_ShortShaFormat) -n 1'
+    <Exec Command='$(GitExe) rev-parse HEAD'

```

Comment:
This is actually a breaking change. The previous line gets the short SHA and assigns it to the GitCommit property. The new code replaces that with the full SHA, which now is exactly the same value as the GitSha property (which is intended to have the long one). 

GitCommit should always have the short sha, and GitSha the long one.

Reply:
[`git rev-parse --short HEAD`](https://git-scm.com/docs/git-rev-parse#_options_for_output)?

Reply:
Update the following to match but for default long format, i.e. without `--short`? https://github.com/devlooped/GitInfo/blob/a2706cf435d9c5f522bd3cb9baa696577c442d43/src/GitInfo/build/GitInfo.targets#L515
You could then remove the `_LongShaFormat` property.

Reply:
Thank you very much @gitfool and @kzu . And I update the code. Could you review my code?

Reply:
looks good